index.php

```diff
@@ -1,131 +1,157 @@
 <?php
 $is_auth = rand(0, 1);
 
-$user_name = ''; // укажите здесь ваше имя
+$user_name = 'Alexandr';
 ?>
 <!DOCTYPE html>
 <html lang="ru">
+
 <head>
     <meta charset="UTF-8">
     <title>Главная</title>
     <link href="../css/normalize.min.css" rel="stylesheet">
     <link href="../css/style.css" rel="stylesheet">
 </head>
-<body>
-<div class="page-wrapper">
 
-<header class="main-header">
-    <div class="main-header__container container">
-        <h1 class="visually-hidden">YetiCave</h1>
-        <a class="main-header__logo">
-            <img src="../img/logo.svg" width="160" height="39" alt="Логотип компании YetiCave">
-        </a>
-        <form class="main-header__search" method="get" action="https://echo.htmlacademy.ru" autocomplete="off">
-            <input type="search" name="search" placeholder="Поиск лота">
-            <input class="main-header__search-btn" type="submit" name="find" value="Найти">
-        </form>
-        <a class="main-header__add-lot button" href="pages/add-lot.html">Добавить лот</a>
-
-        <nav class="user-menu">
+<body>
+    <div class="page-wrapper">
 
-        <!-- здесь должен быть PHP код для показа меню и данных пользователя -->
+        <header class="main-header">
+            <div class="main-header__container container">
+                <h1 class="visually-hidden">YetiCave</h1>
+                <a class="main-header__logo">
+                    <img src="../img/logo.svg" width="160" height="39" alt="Логотип компании YetiCave">
+                </a>
+                <form class="main-header__search" method="get" action="https://echo.htmlacademy.ru" autocomplete="off">
+                    <input type="search" name="search" placeholder="Поиск лота">
+                    <input class="main-header__search-btn" type="submit" name="find" value="Найти">
+                </form>
+                <a class="main-header__add-lot button" href="pages/add-lot.html">Добавить лот</a>
 
-        </nav>
-    </div>
-</header>
+                <nav class="user-menu">
+                    <?php if ($is_auth == 1) : ?>
```

Comment:
```suggestion
                    <?php if ($is_auth === 1) : ?>
```
Правильнее выполнять строгую проверку что бы быть уверенным, что там нужное значение. В твоём случае это не так важно, но лучше сразу привыкать к строгим проверкам, типизации и т.д.
Например, как будет работать нестрогая проверка

```
$var1 = 1;
$var2 = '1';
$var3 = true;

var_dump($var1 == 1); // будет true
var_dump($var2 == 1); // тоже true
var_dump($var3 == 1); // тоже true
```